src/staticwebapp/azext_staticwebapp/_params.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def load_arguments(self, _):
+
+    from azure.cli.core.commands.parameters import resource_group_name_type
+
+    with self.argument_context('staticwebapp dbconnection') as c:
+        c.argument('name', options_list=['--name', '-n'], help="Name of the Static Web App")
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('environment', help="Name of the environment of Static Web App.",
+                   options_list=["--environment", "-e"])
+
+    with self.argument_context('staticwebapp dbconnection create') as c:
+        c.argument('db_resource_id', options_list=['--db-resource-id', '-d'],
+                   help="The azure resource ID for the database server/account to connect to e.g. '/subscriptions/MySubId/resourceGroups/MyResourceGroup/providers/Microsoft.Sql/servers/MyServer' for an Azure SQL database")
+        c.argument('db_name', options_list=['--db-name', '-b'], help="The name of the database to connect to. Not required for CosmosDB.")
+        c.argument('username', options_list=["--username", "-u"], help="The username to use for authentication with the database. Not required for all databases.")
+        c.argument('password', options_list=["--password", "-p"], help="The password to use for authentication with the database. Not required for all databases.")
+        c.argument('mi_user_assigned', options_list=["--mi-user-assigned", "-i"], help="A resource ID for a user-assigned managed identity to use for auth with the database. Must be assigned to the Static Web App and have the right permissions on the database.")
```

Comment:
I'd like to confirm with you that only one user-assigned managed identity is supported for `staticwebapp dbconnection create` at present, right?

Reply:
That's correct, you can provide at most one user-assigned managed identity to be used for the database connections feature